recipe/meta.yaml

```
@@ -7,27 +7,35 @@ package:
   version: {{ version }}
 
 source:
-  url: https://cran.r-project.org/src/base/R-4/R-{{ version }}.tar.gz
-  sha256: b3f5760ac2eee8026a3f0eefcb25b47723d978038eee8e844762094c860c452a
-  patches:
-    - 0001-Darwin-Remove-unicode-elipsis-character-from-grDevic.patch
-    - 0002-Fix-trio-config.h-include-depth-issue.patch
-    - 0003-Win32-Do-not-link-static-libgcc.patch
-    - 0004-Win32-Extend-sqrt-NA_real_-hack-to-all-GCC-versions.patch
-    - 0005-Win32-Prevent-conversion-of-R_ARCH-to-abs-Windows-pa.patch
-    - 0006-Darwin-Avoid-setting-DYLD_FALLBACK_LIBRARY_PATH.patch
-    - 0007-Linux-Do-not-modify-LD_LIBRARY_PATH.patch
-    - 0008-javareconf-Do-not-fail-on-compile-fail.patch
-    - 0009-Revert-part-of-9b818c6dc00143ff18775a4015a3f43b5196f.patch
-    - 0010-javareconf-macOS-Continue-to-allow-system-Java-lt-9-.patch
-    # - 0011-Allow-rpath-in-tcltk-dylib-LC_LOAD_DYLIB-command.patch
-    - 0012-Check-for-changes-then-forcibly-mv-in-javareconf.in.patch
-    - 0013-Use-LAPACK_LDFLAGS-in-Rlapack_la_LIBADD.patch
-    - 0014-Try-forcing-.dylib.patch
+  - url: https://cran.r-project.org/src/base/R-4/R-{{ version }}.tar.gz
+    sha256: b3f5760ac2eee8026a3f0eefcb25b47723d978038eee8e844762094c860c452a
+    patches:
+      - 0001-Darwin-Remove-unicode-elipsis-character-from-grDevic.patch
+      - 0002-Fix-trio-config.h-include-depth-issue.patch
+      - 0003-Win32-Do-not-link-static-libgcc.patch
+      - 0004-Win32-Extend-sqrt-NA_real_-hack-to-all-GCC-versions.patch
+      - 0005-Win32-Prevent-conversion-of-R_ARCH-to-abs-Windows-pa.patch
+      - 0006-Darwin-Avoid-setting-DYLD_FALLBACK_LIBRARY_PATH.patch
+      - 0007-Linux-Do-not-modify-LD_LIBRARY_PATH.patch
+      - 0008-javareconf-Do-not-fail-on-compile-fail.patch
+      - 0009-Revert-part-of-9b818c6dc00143ff18775a4015a3f43b5196f.patch
+      - 0010-javareconf-macOS-Continue-to-allow-system-Java-lt-9-.patch
+      # - 0011-Allow-rpath-in-tcltk-dylib-LC_LOAD_DYLIB-command.patch
+      - 0012-Check-for-changes-then-forcibly-mv-in-javareconf.in.patch
+      - 0013-Use-LAPACK_LDFLAGS-in-Rlapack_la_LIBADD.patch
+      - 0014-Try-forcing-.dylib.patch
```

(I know this is still marked as draft; just leaving this here as a tracking note.)
```suggestion
      - 0014-Try-forcing-.dylib.patch
      - 0018-Fix-path-to-TCL-TK.patch
```